lib/config/validation.ts

```diff
@@ -672,3 +655,48 @@ export async function validateConfig(
   warnings.sort(sortAll);
   return { errors, warnings };
 }
+
+function validateRegexManagerFields(
+  regexManager: RegexManagerConfig,
+  currentPath: string,
+  errors: ValidationMessage[]
+): ValidationMessage[] {
+  if (is.nonEmptyArray(regexManager.matchStrings)) {
+    for (const matchString of regexManager.matchStrings) {
+      try {
+        regEx(matchString);
+      } catch (e) {
+        errors.push({
+          topic: 'Configuration Error',
+          message: `Invalid regExp for ${currentPath}: \`${String(
+            matchString
+          )}\``,
+        });
+        return errors;
```

Comment:
```suggestion
        return;
```
@rarkins should we really stop checks here or also validate the other fields in one go.

Reply:
_It would be best to raise all errors if possible instead of stopping at one_ (( rarkins slack))